spec/requests/authentication_spec.rb

```diff
@@ -74,23 +66,23 @@
       end
 
       context "when caseworker client id is provided" do
-        let(:client_id) { "CASEWORKER" }
+        let(:role) { "Caseworker" }
```

Comment:
is there a reason for the role to be initcapped/titleized

Reply:
I was a bit surprised - the role "values" are lowercase in Entra, but they come out titlecased in the token. Microsoft works in mysterious ways...